src/components/ColorPicker/index.tsx

```diff
@@ -145,7 +145,7 @@ const ColorPicker = ({
     );
     const style = childrenHaveDifferentColors
         ? { border: "1px solid #d3d3d3" }
-        : { backgroundColor: color };
+        : { backgroundColor: lastSelectedColor };
```

Comment:
I actually like the previous behavior where this was `color` instead of `lastSelectedColor`, because the agents in the viewport already update when you change the color. Maybe a question for Lyndsay, but I feel like it makes sense to me for this box to show the current color.